grommet/package.json

```diff
@@ -0,0 +1,148 @@
+{
+  "name": "grommet",
+  "version": "2.4.0",
+  "main": "src/js/index.js",
```

Comment:
This is the only line that was changed in this file.

Reply:
Does `jsnext:main` also need changing?